### DISCUSSION TITLE
Добавление в приложение.

### DISCUSSION THREAD

**Discussion by Joneky** [wheremyfiji/ShikiWatch#6] [General]
Здравствуйте, мне понравилось ваше приложение. Но я хотел бы узнать, собираетесь ли вы в будущем добавить мангу/ранобе тоже?

**Comment by wheremyfiji** [comment]
Спасибо, в будущем точно будет.

**Comment by wheremyfiji** [comment]
Раздел в библиотеке почти готов :)

<img src="https://user-images.githubusercontent.com/26019782/235304936-00a43b1d-7af3-4a96-a080-40e2775f9ca0.jpg" width=40% height=40%>
<img src="https://user-images.githubusercontent.com/26019782/235304930-7404225b-8315-40bb-ba74-8348e7d825ba.jpg" width=40% height=40%>
<img src="https://user-images.githubusercontent.com/26019782/235304938-2a8d3a0c-20a5-4057-9939-7bde524475c1.jpg" width=40% height=40%>
